tests/analysis/direct/test_effectiveness_measures.py

```diff
@@ -102,9 +99,10 @@ def test_init_raises_when_effectiveness_measures_does_not_exist(self):
     )
     def test_knmi_correction_with_invalid_duration_raises(self, duration: int):
         # 1. Define test data.
+        _measures = MockEffectivenessMeasures(None, None)
```

Comment:
```suggestion
        _measures = MockInvalidEffectivenessMeasures(None, None)
```

Reply:
Do you mean creating a different mock for this test, or rename the current mock?

Reply:
Renamed to mock